libbpf-cargo/src/test.rs

```diff
@@ -300,15 +269,13 @@ fn test_build_workspace_collision() {
     let _prog_file_2 = File::create(proj_two_dir.join("src/bpf/prog.bpf.c"))
         .expect("failed to create prog file 2");
 
-    assert_ne!(
-        build(
-            true,
-            Some(&workspace_cargo_toml),
-            Path::new("/bin/clang"),
-            true
-        ),
-        0
-    );
+    build(
+        true,
+        Some(&workspace_cargo_toml),
+        Path::new("/bin/clang"),
```

Comment:
just curious, if you specified "clang", would it try to find it from PATH? Just wondering if it's possible that clang will be in /usr/local/bin/clang, not in /bin/clang.

Reply:
Yes, just "clang" is a path lookup

Reply:
if that's the case, why did you use /bin/clang instead of clang? Sorry if we already discussed this.

Reply:
I suspect it's b/c I put the symlink there: https://github.com/libbpf/libbpf-rs/blob/476712481ddaa2792f27d8bf2a2a8567172d8bb4/.github/workflows/rust.yml#L17

But now that we have more people sending code in it's probably wise to do a PATH search. I'll send a follow up PR